ledger/acctonline.go

```diff
@@ -369,7 +382,9 @@ func (ao *onlineAccounts) prepareCommit(dcc *deferredCommitContext) error {
 	dcc.onlineAccountExpiredRowids = expirations
 	dcc.expirationOffset = expirationOffset
 
-	dcc.genesisProto = ao.ledger.GenesisProto()
+	dcc.genesisProto = config.Consensus[ao.ledger.GenesisProto()]
+
+	dcc.onlineRoundParams = ao.onlineRoundParamsData[offset]
```

Comment:
this does not work, we commit a range from oldBase to oldBase+offset (offset elements), and need to write that range, not a single element